src/clients/mobiles/app.tsx

```diff
@@ -10,6 +11,7 @@ export interface AppProps {
 
 const App = (props: AppProps) => {
     return <Remote {...props} />
+    // return <Tabs {...props} />
```

Comment:
🤔

Reply:
Just a test component working to send Tap action messages.
I wanted to keep it for testing in the future